javascript/new-src/src/controllers/universe.js

```diff
@@ -24,6 +24,7 @@ ola.controller('universeCtrl',
       'use strict';
       $scope.dmx = [];
       $scope.Universe = $routeParams.id;
+      $ola.resetHighestChannelNumberUsed();
```

Comment:
We're better off calling a function here than just initialising a scope variable then are we (like the DMX array)?

Reply:
I'm not very familiar with JavaScript. My home is Java on the backend. :)
I tried to to work with something like this `$scope.highestChannelNumberUsed = 0` but I failed.

Reply:
So it seems like $scope should work if you change it everywhere, but only within one controller, or there's $rootScope, or our $ola might be able to take a variable.

Otherwise @daveol or @FloEdelmann might be able to help, all this whizzy JavaScript library stuff isn't really my area of expertise either unfortunately.